examples/cr.yaml

```diff
@@ -15,12 +15,9 @@ spec:
   # - Unmanaged indicates that the operator should idle and ignore the console and its sub-resources
   # - Removed indicates that the operator should delete all of the sub-resources of the console
   managementState: Managed
-  # there should be only one console at this time
-  # this could be used to support leader election if desired
-  count: 1
+  # there should be the same number of console replicas as the number of masters, minimum 2
```

Comment:
Looks like this (and the PR title and commit message) missed [the bump from 2 to 3][1].

[1]: https://github.com/openshift/console-operator/pull/86#discussion_r239562790

Reply:
Thanks @wking!  Sorry about that, rushed it in for demo sake, should have stopped & updated the extra bits. :/

I'll follow a PR to fix the example file at least.

Reply:
https://github.com/openshift/console-operator/pull/101